source/extensions/common/wasm/null/null_vm.h

```diff
@@ -29,8 +29,7 @@ struct NullVm : public WasmVm {
   bool cloneable() override { return true; };
   WasmVmPtr clone() override;
   bool load(const std::string& code, bool allow_precompiled) override;
-  void link(absl::string_view debug_name, bool needs_emscripten) override;
-  void setMemoryLayout(uint64_t, uint64_t, uint64_t) override {}
+  void link(absl::string_view debug_name) override;
```

Comment:
We are going to have to upstream this.

Reply:
Yup, V8 isn't merged yet, so it's fine I think.